src/app/settings/keyboard-mapping-settings/components/action-selector/action-selector.component.html

```diff
@@ -1,9 +1,18 @@
-<sofie-text-input i18n-label label="action-selector.select-action" i18n-placeholder placeholder="action-selector.search-action" [(value)]="actionSearchQuery"></sofie-text-input>
+<sofie-text-input
```

Comment:
The ActionSelector is _not_ part of the form and should not be marked as required.
It is perfectly valid to not put any data in this input field since you can just scroll the Actions to find your desired Action:
![image](https://github.com/tv2/ng-sofie/assets/23049647/ccab230d-13cc-478f-985f-e4157b15f83f)
What is required however, is the `Selected Action` right beneath the `Trigger On` input:
![image](https://github.com/tv2/ng-sofie/assets/23049647/01a90e6e-4e82-403c-9c27-820e8ed8b7ab)


In short: There should not be validation in the ActionSelector, since the ActionSelector is a tool to fill out the form, not the form itself. But the `Selected Action` should be marked as required.

Reply:
Fixed